sql/src/main/java/io/crate/expression/reference/sys/cluster/ClusterLoggingOverridesExpression.java

```diff
@@ -59,20 +59,9 @@ public ClusterLoggingOverridesExpression(ClusterService clusterService) {
         public static final String NAME = "name";
         public static final String LEVEL = "level";
 
-        protected ClusterLoggingOverridesChildExpression(final Map.Entry<String, String> setting) {
-            childImplementations.put(NAME, new NestableInput<BytesRef>() {
-                @Override
-                public BytesRef value() {
-                    return new BytesRef(setting.getKey());
-                }
-            });
-            childImplementations.put(LEVEL, new NestableInput<BytesRef>() {
-                @Override
-                public BytesRef value() {
-                    return new BytesRef(setting.getValue().toUpperCase(Locale.ENGLISH));
-                }
-            });
-
+        ClusterLoggingOverridesChildExpression(String value) {
+            childImplementations.put(NAME, new LiteralNestableInput<>(new BytesRef(value)));
```

Comment:
afaik this should be the settings key not the value

Reply:
yep, thx - corrected it